.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+image:
+  file: .devcontainer/.gitpod.Dockerfile
+  context: .devcontainer
+
+tasks:
+  - init: >
+      git submodule init
+      && git submodule update
+      && ./autogen.sh
+      && ./configure --enable-ccache --enable-tracy --enable-tracy-capture --enable-tracy-gui
+      && bear make
```

Comment:
I had wondered about that but I also wasn't sure whether parallelism on Gitpod would actually give us any more real hardware CPU time.  It would be a good idea to try, though (if we ever decide to pick this up).